src/models/departments/__tests__/reducer.test.js

```diff
@@ -0,0 +1,63 @@
+import { fromJS } from 'immutable';
+import departmentsJson from 'utils/__tests__/fixtures/departments.json';
+
+import reducer, { initialState } from '../reducer';
+import * as constants from '../constants';
+
+const intermediateState = fromJS({
+  count: 9,
+  error: false,
+  list: departmentsJson.results.slice(0, 9),
```

Comment:
Would suggest using a constant here to make sure the slice and the count use the same number